docs/topics/proc_configure-netty-logging-vertx.adoc

```diff
@@ -0,0 +1,32 @@
+
+[#_configure-netty-logging-vertx]
+= Configuring Netty Logging
+
+link:http://netty.io/wiki/user-guide-for-4.x.html[_Netty_^] is a library used by VertX to manage asynchronous network communication in applications.
+
+Netty:
+
+* allows quick and easy development of network applications, such as protocol servers and clients,
+* simplifies and streamlines network programming, such as TCP and UDP socket server development,
```

Comment:
If you're using bullet points to delimit points in a sentence, you should add _and_ to the end of this point.

Reply:
added, thanks.